api/krusty/openapicustomschema_test.go

```diff
@@ -109,7 +109,7 @@ patchesStrategicMerge:
         - name: server
           image: nginx
 - |-
-  apiVersion: v1alpha1
+  apiVersion: crd.com/v1alpha1
```

Comment:
This test highlighted an interesting scenario. Perhaps an empty `apiVersion` should be treated as a wildcard, but an empty group alone should not?